cli/loader/src/lib.rs

```diff
@@ -807,6 +810,13 @@ impl<'a> LanguageConfiguration<'a> {
                 path_ranges.push((path.clone(), prev_query_len..query.len()));
             }
         } else {
+            if default_path == "highlights.scm" {
```

Comment:
Looks like a hack.

Reply:
Sort of, I didn't do it for other files like tags/injections, maybe we should warn about any defaults being used?